packages/affiliates/libs/bigquery.js

```diff
@@ -0,0 +1,82 @@
+const moment = require("moment");
```

Comment:
yes im not opinionated on the name. can change it to whatever makes the most sense. It could house a lot of different algorithms so it should be pretty generally named.

Reply:
agreed `affiliates` is not really descriptive as it could be and `token-distribution` makes more sense. The `LM` and `gas-rebate` scripts are both "one-off" scripts in the sense that they don't really share much code with the rest of the repo, but functionally they are similar to this package because they query block history to distribute tokens pro-rata